<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkSliderNavigator</class>
 <widget class="QWidget" name="QmitkSliderNavigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>137</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkSliderNavigator</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="3">
    <widget class="ctkDoubleSpinBox" name="m_SpinBox" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="m_SliderLabelLeft">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>32767</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;p align=&quot;center&quot;&gt;XXX&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ctkDoubleSlider" name="m_Slider" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="m_SliderLabelRight">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>45</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>45</width>
       <height>32767</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;p align=&quot;center&quot;&gt;XXX&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>ctkDoubleSlider</class>
   <extends>QWidget</extends>
   <header location="global">ctkDoubleSlider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header location="global">ctkDoubleSpinBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global">mitkStepper.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
